custom-images/init.ps1

```diff
@@ -68,9 +75,6 @@ Set-EnvFileVariable "CM_HOST" -Value "cm.$($HostName).localhost"
 # ID_HOST
 Set-EnvFileVariable "ID_HOST" -Value "id.$($HostName).localhost"
 
-# HRZ_HOST
-Set-EnvFileVariable "HRZ_HOST" -Value "hrz.$($HostName).localhost"
```

Comment:
@AmitKumar-AK If you deleted an "HRZ_HOST" env variable, what about removing host for Horizon on the 134 line:
Add-HostsEntry "hrz.$($HostName).localhost"?

Reply:
Changes, done. Please verify.